docs/manifest-types.md

```diff
@@ -1,5 +1,7 @@
 # Manifest types
 
+[Public documentation]([Public documentation](https://docs.spryker.com/docs/scos/dev/guidelines/keeping-a-project-upgradable/code-upgrader-supported-extension-scenarios.html)) for the section
```

Comment:
I don't understand this sentence to be honest: 

"Public documentation for the section".

Maybe it should be something like "Description of the supported extension scenarios can be found [here]" or something like that.

Reply:
Thanks 
Changed these message